docs/source/integrations/integration-index.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: Apollo Server Integrations
+---
+
+Apollo Server 4 includes two built-in integrations: [`startStandaloneServer`](../api/standalone) and [`expressMiddleware`](../api/express-middleware).
+
+The `startStandaloneServer` function sets useful defaults to get you started quickly. Under the hood, the `startStandaloneServer` function uses Apollo Server 4's Express integration (i.e., `expressMiddleware`). The `expressMiddleware` function is Apollo Server 4's [Express](https://expressjs.com/) integration.
+
+> Are you looking to build a new integration? Or help maintain an existing integration? See [Building Web Framework Integrations for Apollo Server](./building-integrations) for step-by-step guidance!
+
+Apollo and the larger community maintain the following open-source integrations for Apollo Server:
```

Comment:
Apollo does not officially maintain any of these :) (even if some of them are started by Apollo employees)

Reply:
Oh right, thank you! Changed to 

> The larger community maintains the following open-source integrations for Apollo Server:

Do we want to explicitly call out that Apollo doesn't maintain these? Maybe use something like the language in the AS3 data sources articles:

> Apollo does not provide official support for the above community-maintained libraries. We cannot guarantee that community-maintained libraries adhere to best practices or that they will continue to be maintained.